example/tests.py

```diff
@@ -44,12 +47,16 @@ def assert_switch_inactive(browser, url=url):
                 'Switch is not inactive')
 
 
+def drop_datastore():
```

Comment:
It was Switch.c.drop() before, but I guess this is correct. Maybe add a docstring? [shrug]